graph/GuidelinesGraph.md

```diff
@@ -328,6 +329,19 @@ For a complete mapping of error codes to HTTP statuses, see
 
 <a name="api-contract-and-non-backward-compatible-changes"></a>
 
+## External standards
+
+For ease of client use and interoperatibility, some APIs should implement a standard that is defined external to Microsoft Graph and OData. 
+Workloads should follow these standards exactly, even if they conflict with the OData standard and/or the Microsoft Graph guidelines. 
+Workloads must define these standards in their CSDL model if they do not conflict with the OData standard.
+Standards that *do* conflict with the OData standard may be defined in the CSDL in one of two ways:
+1. Using `Edm.Untyped` only and support for the external standard will come directly from the service implementation; OR
+2. Adding CSDL elements to model the external standard using `Edm.String` for `EnumType`s that conflict with the OData standard and `Edm.Untyped` wherever any other conflict with the OData standard occurs
```

Comment:
The structuring of the conjunction makes this sentence seem like it's only about enums

Reply:
The allowed values annotaiton should be used for strings